backend/src/Repository/PackageEntityRepository.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace App\Repository;
+
+use App\Entity\PackageEntity;
+use Doctrine\Bundle\DoctrineBundle\Repository\ServiceEntityRepository;
+use Doctrine\ORM\NonUniqueResultException;
+use Doctrine\Persistence\ManagerRegistry;
+use App\Constant\Package\PackageConstant;
+
+/**
+ * @method PackageEntity|null find($id, $lockMode = null, $lockVersion = null)
+ * @method PackageEntity|null findOneBy(array $criteria, array $orderBy = null)
+ * @method PackageEntity[]    findAll()
+ * @method PackageEntity[]    findBy(array $criteria, array $orderBy = null, $limit = null, $offset = null)
+ */
+class PackageEntityRepository extends ServiceEntityRepository
+{
+    public function __construct(ManagerRegistry $registry)
+    {
+        parent::__construct($registry, PackageEntity::class);
+    }
+
+    /**
+     * @return mixed
+     */
+    public function getActivePackages(): mixed
+    {
+        return $this->createQueryBuilder('package')
+            ->select('package.id, package.name, package.cost, package.note, package.carCount, package.orderCount,
```

Comment:
All SELECT statements of this class can be on the same line because they are not long enough to multiline them.

Reply:
A new line has been created at the end of the screen